chore(deps): upgrade dependencies

aws-cdk-automation opened the pull request:
Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/aws/aws-cdk-cli/actions/runs/14021367402

------

*Automatically created by projen via the "upgrade" workflow*

mrgrain commented:
@kaizencc This seems to have a genuine integ test failure in the csharp tests: https://github.com/aws/aws-cdk-cli/actions/runs/13786170679/job/38555789290#step:13:138

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/aws/aws-cdk-cli/actions/runs/14021367402

------

*Automatically created by projen via the "upgrade" workflow*

Signed-off-by: github-actions <github-actions@github.com>

Commit:
chore: self mutation

Signed-off-by: github-actions <github-actions@github.com>
